File: ATLASDEV/AtlasModel/Enterprise/TTFormFieldEventDef.cs
using System;

namespace AtlasModel.Enterprise
{
    public partial class TTFormFieldEventDef
    {
        public Guid EventDefId { get; set; }
        public Guid FieldDefId { get; set; }
        public string Name { get; set; }
        public string Body { get; set; }
        public short CheckOutStatus { get; set; }
    }
}

File: AtlasWeb/Core/wwwroot/app/NebulaClient/StorageManager/DefinitionManagement/TTFormFieldEventDef.ts
import { Guid } from "../../Mscorlib/Guid";
import { TTFormFieldDef } from "./TTFormFieldDef";

export class TTFormFieldEventDef {
    public EventDefID: Guid;
    public FieldDef: TTFormFieldDef;
    public FieldDefID: Guid;
    public Name: string;
    public Body: string;
    public ToString(): string {
        return null;
    }
}
